templates/graviteeio-apim/0/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+#-------------------------------------------------------------------------------
+# Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#            http://www.apache.org/licenses/LICENSE-2.0
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#-------------------------------------------------------------------------------
+
+elasticsearch:
+  image: elasticsearch:2.3.5
+  hostname: elasticsearch
+  ports:
+    - "8003:9200"
+  expose:
+    - "9300"
+  ulimits:
+    nofile: 65536
+mongodb:
+  hostname: mongodb
+  image: mongo:3.2
+  expose:
+    - "27017"
+
+gateway:
+  image: graviteeio/gateway:${docker_version}
+  hostname: gateway
+  links:
```

Comment:
Set link based in elasticsearch and mongodb services
--      - mongodb
--      - elasticsearch
++      - ${mongodb_service}: mongodb
++      - ${elasticsearch_service}: elasticsearch